surface/model_openapiv3.go

```diff
@@ -44,6 +44,14 @@ func (b *OpenAPI3Builder) buildModel(document *openapiv3.Document) (*Model, erro
 	b.model.Name = document.Info.Title
 	b.model.Types = make([]*Type, 0)
 	b.model.Methods = make([]*Method, 0)
+	for _, v := range document.Components.SpecificationExtension {
+		if v.Name == "x-command-stream-types" {
```

Comment:
Why do we need this check?
Let's just take all extension properties.

Reply:
I made the necessary corrections